compute_tools/src/compute.rs

```diff
@@ -855,6 +855,22 @@ impl ComputeNode {
             info!("Storage auth token not set");
         }
 
+        if let Some(spec) = &compute_state.pspec {
+            match spec.spec.mode {
+                ComputeMode::Primary => {
```

Comment:
Seems like this would be easier if we just did `impl Display for ComputeMode` if we don't already.